drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -923,15 +924,21 @@ local function occupancy_attr_handler(driver, device, ib, response)
 end
 
 local function cumul_energy_exported_handler(driver, device, ib, response)
-  device:set_field(TOTAL_EXPORTED_ENERGY, ib.data.elements.energy.value)
-  device:emit_event(capabilities.energyMeter.energy({ value = ib.data.elements.energy.value, unit = "Wh" }))
+  if ib.data.elements.energy then
```

Comment:
Is this just intended to make sure it is not `nil` for some reason?

Reply:
Yes, since nil + float results an error. In the periodic case it's clear that this may occur since it uses summations, but it may also occur in the cumulative case when `TOTAL_EXPORTED_ENERGY` is used in the PowerConsumptionReport update.